buildSrc/src/main/kotlin/CreateGitTag.kt

```diff
@@ -9,40 +9,39 @@ import org.gradle.work.DisableCachingByDefault
 import javax.inject.Inject
 
 @DisableCachingByDefault(because = "Produces no cacheable output")
```

Comment:
In order to reduce duplication in our codebase, can you make sure that the kotlin version of this file looks the same as this file?

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/main/buildSrc/src/main/kotlin/CreateGitTag.kt

Reply:
c64ccba1917fe678ed255f888bbdbb05df2dcea4